api/src/integrations/core/mine-utils.js

```diff
@@ -101,8 +103,13 @@ class Mines extends BaseRecordUtils {
       throw new Error('getLatestStatus - mineRecord must not be null.');
     }
 
-    const latestStatus = mineRecord.mine_status.pop();
-    return (latestStatus && latestStatus.status_labels.join(' ')) || '';
+    if (!mineRecord.mine_status.length) {
```

Comment:
Do we need a null check for mine_status? Or should it always be non-null?

Reply:
Updated